recipe/meta.yaml

```
@@ -1,19 +1,19 @@
-{% set version = "108" %}
+{% set version = "110" %}
 
 package:
   name: binaryen
   version: {{ version }}
 
 source:
   url: https://github.com/WebAssembly/binaryen/archive/version_{{ version }}.tar.gz
-  sha256: 7fea5013da7f98daeeb6a0a60333e8aa917bcb0b1b418ba8531fe710d09041f2
+  sha256: 0f80250a02b94dd81bdedeae029eb805abf607fcdadcfee5ca8b5e6b77035601
   patches:
     # this fixes an issue where "some toolchains it is necessary to #define __STDC_FORMAT_MACROS before using macros like PRIxPTR"
     # cf: https://github.com/tensorflow/tensorflow/issues/12998#issuecomment-329179887
     - gcc_stdmacro.patch  # [linux]
 
 build:
-  number: 0
+  number: 1
```

Since the version number is bumped, the build number can be set to `0`

```suggestion
  number: 0
```